### DISCUSSION TITLE
Tremor CSS overwrites default Tailwind CSS

### DISCUSSION THREAD

**Discussion by liamseys** [tremorlabs/tremor-npm#267] [Q&A]
![Screenshot 2023-01-05 at 13 10 05](https://user-images.githubusercontent.com/10025985/210777263-9119f02a-cbea-4013-8ad7-2082b3679108.png)

Hi community,

I gave Tremor a first look but it looks when I import the CSS file it overwrites some other styling of my application (Fonts for example). Is there a way to fix this or am I doing anything wrong?

**Comment by mitrotasios** [comment]
Hey @liamseys would you mind sharing a sandbox or a more detailed screenshot that shows which styles are exactly overwritten? It isn't that clear to me from that screenshot :)

**Reply by liamseys** [reply]
![Screenshot 2023-01-06 at 13 26 52](https://user-images.githubusercontent.com/10025985/211012503-5e7c9863-bdb1-4c33-8956-644c725f38a1.png)

Notice how the fonts are thicker when I've imported Tremor's styling? The screenshot in my first post is when Tremor is loaded.

**Reply by mitrotasios** [reply]
sorry, I didn't have time to look into this yet Would you be able to provide a minimal reproducible example of your code by any chance? :)

**Reply by liamseys** [reply]
I've got it fixed, I was importing the CSS after importing TailwindCSS. Thats why some stuff got overwritten. Sorry for the inconvenience!

**Reply by mitrotasios** [reply]
awesome, don't worry, happy to help! This CSS specificity issue is a bit annoying, we'll try to get it fixed in our v2 hopefully 🙏
